xml/deploy_cephadm.xml

```diff
@@ -964,5 +1040,787 @@ ses-min4
 <screen>&prompt.cephuser;ceph orch upgrade stop</screen>
    </sect3>
   </sect2>
+
+  <sect2 xml:id="drive-groups">
+   <title>&drvgrps;</title>
+   <para>
+    <emphasis>&drvgrps;</emphasis> specify the layouts of OSDs in the &ceph;
+    cluster. They are defined in a single YAML file. In this section, we will
+    use <filename>drive_groups.xml</filename> as an example.
+   </para>
+   <para>
+    An administrator should manually specify a group of OSDs that are
+    interrelated (hybrid OSDs that are deployed on solid state and spinners) or
```

Comment:
I think "spinners" is too informal and "solid state" is incomplete. I also note that the 'official' term used later seems to be `rotational`.

How about "hybrid OSDs that are deployed on a mixture of HDDs and SDDs"?